### PR TITLE
Add react-json-tree@0.10.0 w/ npm autoupdate

### LINKED ISSUE #9343: [Request] react-json-tree
React Json Tree
https://github.com/alexkuz/react-json-tree
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/38556567-request-react-json-tree?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #9343 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/alexkuz/react-json-tree
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/react-json-tree
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 388
   - Count of watchers: 7
   - Count of forks: 40
 * NPM download stats (optional):
   - Downloads in the last day: 1,773
   - Downloads in the last week: 40,386
   - Downloads in the last month: 175,583

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9343, cc @alexkuz

### CODE REVIEW

File: tools/fixFormat.js [hunk 1]
```diff
@@ -232,3 +232,4 @@ if (process.argv[1].indexOf('jasmine') == -1) {
 }
 
 exports.fixFormat = fixFormat;
+
```
Comment (PeterDaveHello) [inline tools/fixFormat.js]: You should not touch this one.

**Comment by extend1994** [comment]
@PeterDaveHello please help me check the issue alexkuz/react-json-tree#64, thank you.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by extend1994** [comment]
@PeterDaveHello I think the answer is "No" according to react-json-tree#64. The author has not solved the problem yet. Should we close it now?

**Comment by PeterDaveHello** [comment]
Okay maybe just keep waiting for a few month or something.

**Comment by PeterDaveHello** [comment]
@extend1994 no reason to close?

**Comment by extend1994** [comment]
Accidentally deleted the branch. Tried to restore it but failed in the end. I will open a new pull request for the library when the author solve the issue alexkuz/react-json-tree#64

**Pull request closed without merging**